gprofiler/profiler_state.py

```diff
@@ -0,0 +1,13 @@
+from threading import Event
+
+
+class ProfilerState:
```

Comment:
Cool. Let's add documentation explaining (you can follow my comment where I first suggested it).